core/src/main/java/org/apache/gravitino/meta/ModelVersionEntity.java

```diff
@@ -0,0 +1,174 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.gravitino.meta;
+
+import com.google.common.collect.Maps;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import lombok.ToString;
+import org.apache.gravitino.Auditable;
+import org.apache.gravitino.Entity;
+import org.apache.gravitino.Field;
+
+@ToString
+public class ModelVersionEntity implements Entity, Auditable {
+
+  public static final Field VERSION =
+      Field.required("version", Integer.class, "The version of the model entity.");
+  public static final Field COMMENT =
+      Field.optional("comment", String.class, "The comment or description of the model entity.");
+  public static final Field ALIASES =
+      Field.optional("aliases", List.class, "The aliases of the model entity.");
+  public static final Field URL =
+      Field.required("uri", String.class, "The URI of the model entity.");
+  public static final Field PROPERTIES =
+      Field.optional("properties", Map.class, "The properties of the model entity.");
+  public static final Field AUDIT_INFO =
+      Field.required("audit_info", AuditInfo.class, "The audit details of the model entity.");
```

Comment:
The `ModelVersionEntity` doesn't include the model info?

Does `ModelVersionEntity` not need to include information about `Model`?

Reply:
In my current implementation, it is not needed?

Reply:
if so, how can we determine which `ModelVersionEntity` belongs to which `Model`?